### DISCUSSION TITLE
How to  combine a database as a source and a pdf also as a source, as multiple inputs into pinecone and generate answers to questions?

### DISCUSSION THREAD

**Discussion by babu2020** [FlowiseAI/Flowise#1150] [Q&A]
1. How to  combine a database as a source and a pdf also as a source, as multiple inputs into pinecone and generate answers to questions? 
2. How to feed multiple data sources into pinecone and generate answers to questions?

**Comment by arberrexhepi** [comment]
You do this with seperate flows, or functions, by inserting them to the same index. Then in retrieval you set the index they are stored in.
